README.md

```diff
@@ -12,22 +12,34 @@ Highly Available Tomcat 9 Image
        -> Update your application build.yaml to use this image for hosting your java application 
        -> Update your application deployment.yaml to add the rbac authorizations and roles required to run the tomcat in a cluster and don't forget to add the service account required to run your container under the spec.
 
-#### Note: To run with existing docker configuration, you will not be needed to make any changes to the docker folder
+#### Note: To run with existing docker configuration, you will not be need to make any changes to the docker folder
 
-# How to build image
-## In Openshift
+# Building & Deploying the Image
+## In Openshift (via developer CLI)
 ```
+We tag images using the name of the Jira Issue that caused the image to be built (the SUFFIX parameter).
+Per openshift requirements, we do this in lower-case.
+If you are just running an experiment, you can use your name, your idir, or 'build'.
+
 # NOTE: Switch to target namespace, or set `-n <namespace>` on all `oc` commands
-# oc project ywinub-tools
+# oc project perrsi-tools
 
 # Process template, and create OpenShift resources
-oc -n ywinub-tools process -f openshift/build.yaml SUFFIX=-cvarjao IMAGE_TAG_SUFFIX=-cvarjao | oc -n ywinub-tools create -f -
+oc -n perrsi-tools process -f openshift/build.yaml SUFFIX=oneteam-123 | oc -n perrsi-tools create -f -
+
+# After the first run, use oc update instad of oc create to update the existing image.
```

Comment:
typo - instead. And may be worthwhile to provide the complete command too?

Reply:
Fixed the typo, and copied in the command - readme bytes are cheap!